cmake/kconfig.cmake

```diff
@@ -30,15 +34,18 @@ set(CREATE_NEW_DOTCONFIG "")
 if(NOT EXISTS ${DOTCONFIG} OR ${BOARD_DEFCONFIG} IS_NEWER_THAN ${DOTCONFIG})
   set(CREATE_NEW_DOTCONFIG 1)
 elseif(${CONF_FILE})
-  if(${CONF_FILE} IS_NEWER_THAN ${DOTCONFIG})
-    set(CREATE_NEW_DOTCONFIG 1)
-  endif()
+  foreach(CONF_FILE_ITEM ${CONF_FILE_AS_LIST})
+    if(${CONF_FILE_ITEM} IS_NEWER_THAN ${DOTCONFIG})
+      set(CREATE_NEW_DOTCONFIG 1)
+    endif()
+  endforeach()
 endif()
 if(CREATE_NEW_DOTCONFIG)
   execute_process(
     COMMAND ${PYTHON_EXECUTABLE} ${PROJECT_SOURCE_DIR}/scripts/kconfig/merge_config.py -m -q
       -O ${PROJECT_BINARY_DIR}
-      ${BOARD_DEFCONFIG} ${CONF_FILE}
+      ${BOARD_DEFCONFIG}
+      ${CONF_FILE_AS_LIST}
```

Comment:
Awesome, thanks for the update.

Shouldn't this be `${CONF_FILE}`? If there are multiple files, CONF_FILE_AS_LIST is semicolon separated, not whitespace separated.